package.json

```diff
@@ -93,8 +93,9 @@
     "scripts"
   ],
   "overrides": {
-    "@types/enzyme": {
-      "@types/react": "^17.0.5"
-    }
+    "@types/react": "^17.0.83",
+    "@types/react-dom": "^17.0.25",
+    "react": "^17.0.2",
+    "react-dom": "^17.0.2"
```

Comment:
This is here mostly so that we can force outdated libraries like enzyme, ag-grid, reflux to align on the version of react and react types that we are using. We can in theory scope it only to those packages, but ultimately this is the version of react we expect to be running the application against, it's forced in runtime by our webpack config, so it doesn't sound like a bad idea to me to also force this on all the packages we install with this override